src/gui/widgets/LcdWidget.cpp

```diff
@@ -176,21 +185,24 @@ void LcdWidget::paintEvent( QPaintEvent* )
 	}
 
 	// Right Margin
-	p.drawPixmap( QRect( 0, 0, m_marginWidth-1, m_cellHeight ), *m_lcdPixmap, 
-			QRect( charsPerPixmap*m_cellWidth, isEnabled()?0:m_cellHeight,
-				m_cellWidth / 2, m_cellHeight ) );
+	p.drawPixmap(QRect(0, 0, m_seamlessRight ? m_marginWidth : m_marginWidth - 1, m_cellHeight),
+		*m_lcdPixmap,
+		QRect(charsPerPixmap * m_cellWidth, isEnabled() ? 0 : m_cellHeight, m_cellWidth / 2, m_cellHeight));
```

Comment:
I played with a few suggestions to make this a little more readable as those inline tertiary operators could be hard to parse at a quick glance, but I'm just not sure what would be best here or just ignore it and let it be an eyesore 😄.